contracts/BaseStrategy.sol

```diff
@@ -732,38 +749,33 @@ abstract contract BaseStrategy {
      * @return `true` if `harvest()` should be called, `false` otherwise.
      */
     function harvestTrigger(uint256 callCostInWei) public view virtual returns (bool) {
-        uint256 callCost = ethToWant(callCostInWei);
-        StrategyParams memory params = vault.strategies(address(this));
+        // Should not trigger if strategy is not active (no assets or no debtRatio)
+        if (!isActive()) return false;
+
+        // check if the base fee gas price is higher than we allow. if it is, block harvests.
+        if (!isBaseFeeAcceptable()) return false;
 
-        // Should not trigger if Strategy is not activated
-        if (params.activation == 0) return false;
+        // trigger if we want to manually harvest, but only if our gas price is acceptable
+        if (forceHarvestTriggerOnce) return true;
```

Comment:
The idea is that by calling `forceHarvestTriggerOnce` you are still respecting the max base fee acceptable—a situation where you want to harvest it but gas is currently too high, you you flip the boolean and know it will harvest later when optimal.

Reply:
If management/gov really needs to harvest immediately, they can either atomically update the baseFeeOracle so that the gas price check will be permissible, or just simply call `harvest()` themselves

Reply:
> In which case does the management changes `forceHarvestTriggerOnce` and not call `harvest` themselves?

management would set `forceHarvestTriggerOnce` to `true` when they want keepers to harvest the next time when gas is optimal, instead of doing it now manually themselves when gas may be more expensive. 

additionally, since we use keep3r on mainnet, we don't really "pay" for harvests done by keepers since we already have the LPs bonded, but if we call harvest from the multisig then we have to spend additional ETH on it.